assets/css/style.css

```diff
@@ -0,0 +1,1262 @@
+    align-items: center;
+    gap: 1rem;
+    height: 10rem;
+  }
+
+  .about-footer1 img {
+    width: 30%;
+  }
+
+  .about-footer1 p {
+    font-family: "Lato", sans-serif;
+    font-size: 1rem;
+    color: #837f7f;
+  }
+
+  #mlist li {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 70;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff5e1;
+    margin-left: 40%;
+    list-style-type: none;
+    margin-top: 10px;
+    margin-bottom: 5px;
+  }
+
+  #menu- {
+    background-color: #837f7f;
+    height: 450px;
+    margin-top: -120px;
+    display: none;
+    z-index: 1;
+  }
+
+  #closed {
+    margin-left: 25px;
+    color: #fff;
+    font-size: 60px;
+    margin-top: 80px;
+  }
+
+  #mlist li a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  #menu-list hr {
+    color: #6f6c6b;
+    background-color: #6f6c6b;
+    width: 90%;
+    height: 1px;
+    margin-left: 24px;
+    right: 0;
+    bottom: 0;
+  }
+
+  .See-the-whole,
+  .whole-program,
+  .partners,
+  .footer,
+  .top-header,
+  .header-desk,
+  #btn-less,
+  .footerdesk-,
+  .partner,
+  .about-footer,
+  .footerdesk1 {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .top-header {
+    background-color: #3e3e3e;
+    height: 3rem;
+    display: flex;
+    justify-content: end;
+    color: #f4f4f4;
+  }
+
+  .top-header ul {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    margin-right: 5rem;
+  }
+
+  .partner li {
+    font-family: "Lato", sans-serif;
+    font-size: 3em;
+    margin: 1em 0.3em;
+    color: #a9aaaa;
+    font-weight: 200;
+    margin-top: 0%;
+  }
+
+  .top-header ul li {
+    list-style: none;
+    font-family: "Lato", sans-serif;
+    font-size: 6px;
+  }
+
+  .top-header ul li .top-icon {
+    height: 1.5rem;
+    width: 1.5rem;
+    cursor: pointer;
+    margin-top: 0.4rem;
+  }
+
+  .header-desk {
+    background-color: #fff;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .navbar {
+    display: block;
+    position: relative;
+    transform: none;
+    opacity: inherit;
+    visibility: visible;
+    pointer-events: auto;
+    width: 100%;
+    margin-right: 10%;
+  }
+
+  .nav-links {
+    flex-direction: row;
+    justify-content: flex-end;
+    gap: 2rem;
+    display: flex;
+    list-style: none;
+    margin-top: 15px;
+    text-decoration: none;
+    position: relative;
+    width: 100%;
+  }
+
+  .nav-item {
+    border-bottom: 0;
+    text-decoration: none;
+  }
+
+  .nav-item a {
+    text-decoration: none;
+    color: #21212c;
+    font-size: 10px;
+  }
+
+  .logo {
+    display: block;
+    margin-left: 5rem;
+  }
+
+  .logo img {
+    width: 150px;
+    height: 40px;
+  }
+
+  .header {
+    background: url(../images/bg-image.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 400px;
+    box-shadow: inset 0 0 0 1000px rgba(210, 210, 209, 0.8);
+  }
+
+  .hero-content {
+    margin-left: 20px;
+  }
```

Comment:
- [ ] Please make this container aligned to its right place by adding the true spacings. Your project seems depressed and not matching the given design. The section should cover the screen height as can be seen below:
yours: 
![image](https://user-images.githubusercontent.com/91486791/203618662-48154a30-edb0-443e-adda-d5ccb06434a1.png)
figma:
![image](https://user-images.githubusercontent.com/91486791/203618908-7f369ed4-35d1-4657-9453-6ff1bb230d7b.png)